backend/tenant_account/users_view.py

```diff
@@ -32,7 +32,6 @@ def assign_organization_role_to_user(self, request: Request) -> Response:
             raise BadRequestException
         org_id: str = UserSessionUtils.get_organization_id(request)
         auth_controller = AuthenticationController()
-
         auth_controller = AuthenticationController()
```

Comment:
@vishnuszipstack  I don't think this is related to your change observing multiple initializations of Authcontroller.

Reply:
removed @johnyrahul